azure-pipelines-dependency-graph-testing.yml

```diff
@@ -358,7 +358,7 @@ stages:
           - script: |
               # These are tests that are specific to Great Expectations Cloud.
               # In order to ensure coverage, we run them during each CI/CD cycle.
-              pytest tests -m cloud --cloud
+              pytest tests --cloud -m "cloud and not integration"
```

Comment:
Does it know operands like "and" and "not" or is this just the name of a pipeline?

Reply:
yeah `pytest` syntax is weird but per the docs, it allows for boolean operators like "and"/"or"